test/runtests.jl

```diff
@@ -159,7 +159,7 @@ end
 # end
 
 # Test against MKL_jll using `libmkl_rt`, which is :LP64 by default
-if MKL_jll.is_available()
+if MKL_jll.is_available() && !Sys.iswindows()
```

Comment:
Why are you limiting this to non-windows?

Reply:
I got an issue with rights ln `ibmkl_rt.dll` but it seems to be fixed with the latest JLL (2024.2.0).

Reply:
I believe that was the first MKL release where all our known issues got sorted out.